roles/controller_casc_from_aap/tasks/inventory_sources_from_aap_improved.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: "Get current Inventory Source from the API"
+  set_fact:
+    inventory_source_lookvar: "{{ query('ansible.controller.controller_api', 'api/v2/inventory_sources/', host=controller_hostname, username=controller_username, password=controller_password, verify_ssl=controller_validate_certs, return_all=true) }}"
+
+- name: "Create the output directory: {{ output_path }}"
+  file:
+    path: "{{ output_path }}"
+    state: directory
+
+- name: "Add current inventory source to the output yaml file: {{ output_path }}/current_inventory_sources.yaml"
+  template:
+    src: "templates/current_inventory_sources_improved.j2"
+    dest: "{{ output_path }}/current_inventory_sources.yaml"
+  vars:
+    current_inventory_source_asset_value: "{{ inventory_source_lookvar }}"
```

Comment:
This should be current_inventory_sources_asset_value

Reply:
changed!